definitions/npm/bluebird_v3.x.x/flow_v0.32.x-/bluebird_v3.x.x.js

```diff
@@ -142,6 +142,7 @@ declare class Bluebird$Promise<R> {
   each<T, U>(iterator: (item: T, index: number, arrayLength: number) => Bluebird$Promisable<U>): Bluebird$Promise<Array<T>>;
   asCallback<T>(callback: (error: ?any, value?: T) => any, options?: Bluebird$SpreadOption): void;
   return<T>(value: T): Bluebird$Promise<T>;
+  spread(...args: Array<any>): Bluebird$Promise<any>;
```

Comment:
Mostly because the promise can resolve with many values of different types, e.g. a string and an object. Would a generic still be correct? (I'm not too well versed in flow generics, but that'd be wrong in Java)